app/views/account/workspaces/show.html.slim

```diff
@@ -1,3 +1,22 @@
-- title @workspace.name
-.title
-  = @workspace.name
\ No newline at end of file
+.container
+  .level
+    .level-left
+      .title
```

Comment:
don't you need to add "level-item" here http://prntscr.com/lbheqx ?
as below, looks like in bulma it's required